src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -98,6 +98,12 @@ internal sealed class TupleTypeSymbol : WrappedNamedTypeSymbol
 
             NamedTypeSymbol underlyingType = GetTupleUnderlyingType(elementTypesWithAnnotations, syntax, compilation, diagnostics);
 
+            if (numElements >= RestPosition && diagnostics != null && !underlyingType.IsErrorType())
+            {
+                WellKnownMember wellKnownTupleRest = GetTupleTypeMember(RestPosition, RestPosition);
+                _ = (FieldSymbol)GetWellKnownMemberInType(underlyingType.OriginalDefinition, wellKnownTupleRest, diagnostics, syntax);
```

Comment:
>GetWellKnownMemberInType [](start = 33, length = 24)

Is this to report use-site diagnostics? Aren't those reported by `GetTupleUnderlyingType()`?

Reply:
I think that checks all tuple fields. Is there a reason we would need to mandate all fields are present? Not having Rest breaks definite assignment, so that one's important.